qemu/tests/cfg/qemu_img_convert_with_rate_limit.cfg

```diff
@@ -0,0 +1,39 @@
+- qemu_img_convert_with_rate_limit:
+    virt_test_type = qemu
+    type = qemu_img_convert_with_rate_limit
+    start_vm = no
+    kill_vm = yes
+    force_create_image = no
+    guest_temp_file = "/var/tmp/convert.tmp"
```

Comment:
just fyi.
/tmp/convert.tmp might be good, for we don't have to remove it post test.
But it's OK to use /var/tmp/xxx, cause the file is very small, and it is always convert.tmp created when running the tc.

Reply:
just fyi.
Create a new file and check its md5sum to ensure a successful converting image, what about handle it in a simple way, start vm with converted image, and we can log into the vm.

Reply:
Thanks zhenchao. I change to /tmp/convert.tmp. For the steps start vm with converted image and log into it to check file md5sum are contained in my test.